docs/content/preview/yugabyte-cloud/cloud-monitor/metrics-export.md

```diff
@@ -43,16 +48,39 @@ You can add, edit, and delete export configurations. You can't delete a configur
 
 To create an export configuration, do the following:
 
+{{< tabpane text=true >}}
+
+  {{% tab header="Datadog" lang="datadog" %}}
+
 1. On the **Integrations** page, select the **Metrics** tab.
 1. Click **Create Export Configuration** or, if one or more configurations are already available, **Add Export Configuration**.
 1. Enter a name for the configuration.
-1. Choose your provider. (Currently, only Datadog is supported.)
+1. Choose Datadog.
 1. Enter your Datadog [API key](https://docs.datadoghq.com/account_management/api-app-keys/).
 1. Choose the Datadog site to connect to, or choose Self-hosted and enter your URL.
 1. Optionally, click **Download** to download the Datadog dashboard template. You can import this JSON format template into your Datadog account and use it as a starting point for visualizing your cluster data in Datadog.
 1. Click **Test Configuration** to make sure your connection is working.
 1. Click **Create Configuration**.
 
+  {{% /tab %}}
+
+  {{% tab header="Grafana" lang="grafana" %}}
+
+1. On the **Integrations** page, select the **Metrics** tab.
+1. Click **Create Export Configuration** or, if one or more configurations are already available, **Add Export Configuration**.
+1. Enter a name for the configuration.
+1. Choose Grafana.
+1. Enter your organization name. This is displayed in the URL when you connect to your Grafana Cloud Portal (for example, `https://grafana.com/orgs/<organization-name>`).
+1. Enter your Grafana [Access policy token](#grafana).
+1. Enter your Grafana instance ID and zone. Obtain these by navigating to the Cloud portal, selecting your stack, and on the Grafana tile, clicking **Details**. They are displayed under **Instance Details**.
+1. Optionally, click **Download** to download the Grafana dashboard template. You can import this JSON format template into your Grafana account and use it as a starting point for visualizing your cluster data in Grafana.
```

Comment:
From where do I click "Download"? The same Export Configuration page ? As long as it seems obvious to a user, ignore my question. I'm asking because I can't see the UI and verify currently.